src/components/Search/SearchOverTime.tsx

```diff
@@ -0,0 +1,178 @@
+import { useEffect, useState, useCallback } from 'react';
+
+import InfoIcon from '@mui/icons-material/InfoOutlined';
+import Grid from '@mui/material/Grid';
+import InputLabel from '@mui/material/InputLabel';
+import Tooltip from '@mui/material/Tooltip';
+import { cssRule } from 'typestyle';
+
+import { compareView } from '../../common/constants';
+import { useAppSelector } from '../../hooks/app';
+import {
+  Spacing,
+  DropDownMenuRaw,
+  DropDownItemRaw,
+  //SearchStyles can be found in CompareCards.ts
+  SearchStyles,
+} from '../../styles';
+import { RevisionsList } from '../../types/state';
+import SearchDropdown from './SearchDropdown';
+import SearchInput from './SearchInput';
+import SearchResultsList from './SearchResultsList';
+
+interface SearchProps {
+  selectLabel: string;
+  searchResults: RevisionsList[];
+  tooltip: string;
+  inputPlaceholder: string;
+  isEditable: boolean;
+}
+
+function SearchComponent({
+  selectLabel,
+  tooltip,
+  inputPlaceholder,
+  isEditable,
+  searchResults,
+}: SearchProps) {
+  const mode = useAppSelector((state) => state.theme.mode);
+  const styles = SearchStyles(mode);
+  const [displayDropdown, setDisplayDropdown] = useState(false);
+  //temporary until next PR covers selected revisions
+  const handleSearchResultsEditToggle = (toggleArray: RevisionsList[]) => {
+    console.log('handleSearchResultsEditToggle', toggleArray);
+  };
+
+  const displayedRevisions = {
+    revs: [],
+    repos: [],
+  };
+
+  /* These overriding rules update the theme mode by accessing the otherwise inaccessible MUI tooltip styles */
+  cssRule('.MuiPopover-root', {
+    $nest: {
+      '.MuiPaper-root': {
+        flexDirection: 'column',
+        ...(mode === 'light' ? DropDownMenuRaw.Light : DropDownMenuRaw.Dark),
+        $nest: {
+          '.MuiList-root': {
+            padding: `${Spacing.Small}px ${Spacing.xSmall}px`,
+            $nest: {
+              '.MuiMenuItem-root': {
+                ...(mode === 'light'
+                  ? DropDownItemRaw.Light
+                  : DropDownItemRaw.Dark),
+              },
+            },
+          },
+        },
+      },
+    },
+  });
```

Comment:
I'm sure there are better ways ot handle that (for example MUI's `styleOverrides`), because contrary to the other override in  `styles`, this override here should probably be applied more broadly.
I'm not asking to solve this now, but it would be good to spend some time on this question later on.

Especially I'm not comfortable that we add these globally generic rules inside a component, and they can apply in code that is used in other components.

I'd love to hear what you think, because I'm sure you know more than me about this library.

Reply:
I'll create a ticket to take care of this work. It needs an update.